nginx/etc/nginx/nginx.conf

```diff
@@ -0,0 +1,136 @@
+user nginx;
+worker_processes 4;
```

Comment:
I think we should make this 1 and make the number of task definitions the unit of concurrency.